easybuild/easyconfigs/m/MaSuRCA/MaSuRCA-3.3.0-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,60 @@
+##
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2017 University of Geneva
+# Authors::   Yann Sagon <yann.sagon@unige.ch>
+# License::   MIT/GPL
+# $Id$
+#
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'MaSuRCA'
+version = '3.3.0'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://www.genome.umd.edu/masurca.html'
+
+description = '''MaSuRCA is whole genome assembly software. It combines the efficiency of the de Bruijn graph  
+ and Overlap-Layout-Consensus (OLC) approaches. MaSuRCA can assemble data sets containing 
+ only short reads from Illumina sequencing or a mixture of short reads and long reads 
+ (Sanger, 454, Pacbio and Nanopore).'''
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+# note: the tagged 3.3.0 release is (slightly) older than the 3.3.0 tarball included *in* the repo...
+source_urls = ['https://github.com/alekseyzimin/masurca/raw/9b3771/']
+sources = ['%(name)s-%(version)s.tar.gz']
+checksums = ['f4625de837c54311d2ff4a53cd7e60d28bae49c1e5e9f2c1cbd75b688454d078']
+
+dependencies = [
+    ('libreadline', '7.0'),
+    ('Tcl', '8.6.8'),
+    ('Boost', '1.67.0'),
+    ('zlib', '1.2.11'),
+    ('Perl', '5.28.0'),
```

Comment:
Doesn't it need bzip2 any longer? the 3.2 version had bzip2 as dependency.

Reply:
added `bzip2` in 05057341f after checking that it's indeed requires (the script call `bzip2`)